### ISSUE TITLE
"react/wrap-multilines" opposite

### ISSUE THREAD

**Issue by SeeThruHead** [yannickcr/eslint-plugin-react#196]
Is it possible to add a rule to enforce no parens in multiline jsx.

**Comment by yannickcr** [comment]
The [no-extra-parens](http://eslint.org/docs/rules/no-extra-parens) eslint rule should warn you about parens around multiline jsx.

But a specific rule for this could also be added.

**Comment by michaelBenin** [comment]
I'm actually running into the opposite.

I'd like to enforce parens in the return statement.

For example:

```
render() {
    return (
      <html>
      <Head />
      <body>
      <Header />
      <main className="main" role="content">
        {this.props.children}
      </main>
      <Footer />
      <script src="//localhost:35729/livereload.js" async></script>
      <script src="//localhost:3001/static/bundle.js" async></script>
      </body>
      </html>
    );
  }
```

This way my html stays nicely indented and readable.

**Comment by yannickcr** [comment]
@michaelBenin yes, no-extra-parens and react/wrap-multilines cannot cannot be enabled at the same time.

More details in #86

**Comment by ljharb** [comment]
I think this is already achievable; if not, please open a new issue :-)
